File: vs_semester-5/oop_lab-2/task-2_cs/City.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace task_2_cs {
  class City : Metropolis {
    public City(string cityName) : base("unknown") {
      this.cityName = cityName;
    }
    protected override void getInfo() {
      Console.WriteLine(domainName);
      Console.WriteLine(metropolisName);
      Console.WriteLine(cityName);
      Console.WriteLine();
    }
    protected string cityName;
  }
}

File: course-3_semester-5/java/oop_task-2/City.java
class City extends Metropolis {
  public City(String cityName) {
    super("unknown");
    this.cityName = cityName;
  }
  protected void getInfo() {
    System.out.println(domainName);
    System.out.println(metropolisName);
    System.out.println(cityName);
    System.out.println();
  }
  protected String cityName;
}